pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -178,9 +178,6 @@ def _get_max_batches(self) -> List[int]:
             max_batches = self.trainer.num_test_batches
         else:
             if self.trainer.sanity_checking:
-                self.trainer.num_sanity_val_batches = [
-                    min(self.trainer.num_sanity_val_steps, val_batches) for val_batches in self.trainer.num_val_batches
-                ]
```

Comment:
Why is this moved?

Reply:
no reason, just for better readability. since setting it inside `_get_max_batches` doesn't makes sense.

Reply:
Sounds good to me if it works. I remember I tried moving this a long time ago but the structure did not allow it. Maybe it's different now